client/src/main/java/de/rebleyama/client/RebleyamaClient.java

```diff
@@ -1,14 +1,26 @@
 package de.rebleyama.client;
 
-import com.badlogic.gdx.*;
-import com.badlogic.gdx.graphics.*;
+import com.badlogic.gdx.ApplicationAdapter;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.InputMultiplexer;
+import com.badlogic.gdx.InputProcessor;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.OrthographicCamera;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
-import com.badlogic.gdx.maps.tiled.*;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TiledMapRenderer;
+import com.badlogic.gdx.maps.tiled.TmxMapLoader;
 import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
-
+import com.badlogic.gdx.scenes.scene2d.Stage;
 
 public class RebleyamaClient extends ApplicationAdapter implements InputProcessor {
 
+    private static final int TILESIZE = 40;
+    private static final int MAPSIZE_TILES = 512;
```

Comment:
see above

Reply:
This is a temporary constant and should eventually be moved to a config file or similar. I don't see the value in changing this right now.